src/test/java/com/FindDeadLinksTest.java

```diff
@@ -22,6 +22,24 @@
     private String expected = "";
 
 
+    @Before
+    public void setExpectedOutputFromJsonFile() {
```

Comment:
@Richardowsky isn't this a duplication of https://github.com/ekondrashev/java-dead-links/pull/51/files#diff-24525881226c0d16f4bbb5256963872eR44